app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -2,7 +2,6 @@ module Katello
   class Api::V2::UpstreamSubscriptionsController < Api::V2::ApiController
     before_action :find_organization
     before_action :check_upstream_connection
-    before_action :deprecated, only: [:simple_content_access_eligible, :enable_simple_content_access, :disable_simple_content_access]
```

Comment:
We just added these deprecations so unfortunately it's too soon to remove them!

Reply:
We should adjust the annotation in the deprecation method `it will be removed in Katello 4.0`.